plugins/arm_a64/a64_defs.ml

```diff
@@ -0,0 +1,213 @@
+open Core_kernel
```

Comment:
We don't really need this module at all (or at least its current contents). There is no need to maintain two sets of definitions for registers as variables and as polymorphic constructors. This is a design error inherited from the ARMv7 lifter.

Reply:
btw, by this module I didn't mean `Core_kernel` as it may look like, it was a comment to the whole file, i.e., `a64_defs.ml`